output.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cobra2snooty
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	outputHeader = `Output
+------
+`
+)
+
+// This function can return the output for all commands when the output template is added as an annotation in the command file
+
+func printOutputCreate(buf *bytes.Buffer, cmd *cobra.Command) {
+	removerange := strings.ReplaceAll(cmd.Annotations["output"], "{{range .Results}}", "")
+	removeend := strings.ReplaceAll(removerange, "{{end}}", "")
+	bracketsremoved1 := strings.ReplaceAll(removeend, "{{.", "<")
+	bracketsremoved2 := strings.ReplaceAll(bracketsremoved1, "}}", ">")
+	replacevariable := strings.ReplaceAll(bracketsremoved2, "%s", "<Name>")
+	replacevariable2 := strings.ReplaceAll(replacevariable, "\n", "\n   ")
+	w := new(tabwriter.Writer)
+	w.Init(buf, 0, 0, 0, ' ', tabwriter.Debug|tabwriter.AlignRight)
```

Comment:
not sure if this matters but these are the values we use for the tabwritter on the cli
https://github.com/mongodb/mongodb-atlas-cli/blob/7a304c4870c026d80e5b26391efad0e2c48f0f26/internal/templatewriter/templatewriter.go#L23-L34

Reply:
I experimented with these a bit and made some changes to the code overall to reflect a different alignment and padding for this. The thing that has me stumped right now: when I add 3 spaces to the beginning of the first line for these, it messes up the alignment of all other headers by 3 spaces. But without the 3 spaces it won't be in the code box.
**With 3 extra spaces at the beginning: ID aligns correctly, others are 3 spaces too far in:**
![Screenshot 2023-03-14 at 9 27 46 AM](https://user-images.githubusercontent.com/82042374/225016101-98fcfc57-2ac4-4d5f-9f20-ffe1fabe4e10.png)
**Without 3 extra spaces at the beginning: All except ID align correctly:**
![Screenshot 2023-03-14 at 9 31 34 AM](https://user-images.githubusercontent.com/82042374/225016738-f717f6e1-53dc-41ff-bf90-86900a3d76e0.png)